<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnalysisSettingsConfirmationDialog</class>
 <widget class="QDialog" name="AnalysisSettingsConfirmationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>885</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget_contents" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_header">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <bold>true</bold>
          <underline>true</underline>
         </font>
        </property>
        <property name="text">
         <string>Attention: Only the following speakers and variables will be considered for the analysis!</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_speakers">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Speakers:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget_speakerlist">
        <property name="styleSheet">
         <string notr="true">QListWidget {
	font-size: 16px;
}</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_dvs">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Dependent Variables:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget_dvlist">
        <property name="styleSheet">
         <string notr="true">QListWidget {
	font-size: 16px;
}</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_btns" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btn_cancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_includeall">
        <property name="text">
         <string>Discard changes...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_savesettings">
        <property name="text">
         <string>Save settings...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_cancel</sender>
   <signal>clicked()</signal>
   <receiver>AnalysisSettingsConfirmationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>370</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_savesettings</sender>
   <signal>clicked()</signal>
   <receiver>AnalysisSettingsConfirmationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>492</x>
     <y>329</y>
    </hint>
    <hint type="destinationlabel">
     <x>492</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
